tony-azkaban/src/test/java/com/linkedin/tony/azkaban/TestTonyJob.java

```diff
@@ -63,6 +70,48 @@ public String getWorkingDirectory() {
     Assert.assertTrue(args.contains(TonyJobArg.SHELL_ENV.tonyParamName + " E1=e1"));
   }
 
+  /**
+   * Check if the flow level information is passed to the tony job through configuration.
+   */
+  @Test
+  public void testFlowInfoPropagation() {
+    final Props jobProps = new Props();
+    jobProps.put(TonyJobArg.HDFS_CLASSPATH.azPropName, "hdfs://nn:8020");
+    jobProps.put(CommonJobProperties.PROJECT_NAME, "unit_test");
+    jobProps.put(CommonJobProperties.FLOW_ID, "1");
+    jobProps.put(CommonJobProperties.EXEC_ID, "0");
+
+    final TonyJob tonyJob = new TonyJob("test_tony_job", new Props(), jobProps, log) {
+      @Override
+      public String getWorkingDirectory() {
+        return System.getProperty("java.io.tmpdir");
+      }
+    };
+    tonyJob.getMainArguments();
+
+    org.apache.hadoop.conf.Configuration conf =
```

Comment:
Add a `import org.apache.hadoop.conf.Configuration` rather than using the full class name here.

Reply:
Fixed